bunsen/bunsen-core-r4/src/main/java/com/cerner/bunsen/definitions/r4/R4StructureDefinitions.java

```diff
@@ -306,6 +307,11 @@ private <T> List<StructureField<T>> elementToFields(DefinitionVisitor<T> visitor
 
       return extensionElementToFields(visitor, rootDefinition, element, definitions, stack);
 
+    } else if (element.getSliceName() != null) {
+
+      // Drop slices for non-extension fields; otherwise we will end up with duplicated fields.
+      return Collections.emptyList();
```

Comment:
Just wanted to confirm that have we covered this case from unit test?

Reply:
Yes, the new `medicationRequestConverter` which is based on US Core MedicationRequest, triggers this path (it defines a slice on `category`). It is used in the uncommented `testContainedResources`. That's actually how I found out about the slicing bug.

Reply:
thanks